src/data/real/cardinality.lean

```diff
@@ -2,20 +2,48 @@
 Copyright (c) 2019 Floris van Doorn. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Floris van Doorn
-
-The cardinality of the reals.
 -/
 import set_theory.cardinal_ordinal
 import analysis.specific_limits
 import data.rat.denumerable
 import data.set.intervals.image_preimage
+/-!
+# The cardinality of the reals
+
+This file shows that the real numbers have cardinality continuum, i.e. `#ℝ = 2^ω`.
+
+We shows that `#ℝ ≤ 2^ω` by noting that every real number is determined by a Cauchy-sequence of the
+form `ℕ → ℚ`, which card cardinality `2^ω`. To show that `#ℝ ≥ 2^ω` we define an injection from
+`{0, 1} ^ ℕ` to `ℝ` with `f ↦ Σ n, f n * (1 / 3) ^ n`.
+
+We conclude that all intervals with distinct endpoints have cardinality continuum.
+
+## Main definitions
 
+* `cantor_function` is the function that sends `f` in `{0, 1} ^ ℕ` to `ℝ` by
```

Comment:
```suggestion
* `cardinal.cantor_function` is the function that sends `f` in `{0, 1} ^ ℕ` to `ℝ` by
```
With a full namespace it will be linkified in the docs